birdman/model_base.py

```diff
@@ -257,40 +335,62 @@ def to_inference_object(
         if self.fit is None:
             raise ValueError("Model has not been fit!")
 
+        # if already Inference, just return
+        if isinstance(self.fit, az.InferenceData):
+            return self.fit
+        if isinstance(self.fit, list):
+            if isinstance(self.fit[0], az.InferenceData):
+                return self.fit
```

Comment:
I'd think we'd want to concat these objects together right?  If so, the following will do
```suggestion
                cat_name = self.specifications["concatenation_name"]
                coords = self.specifications["coords"] 
                return concatenate_inferences(self.fit, coords, cat_name)
```

Reply:
Good catch. I think here it makes sense to check if `combine_individual_fits == True` and then proceed accordingly.